build_tools/fish_xgettext.fish

```diff
@@ -2,84 +2,90 @@
 #
 # Tool to generate messages.pot
 
-# Create temporary directory for these operations. OS X `mktemp` is somewhat restricted, so this block
-# works around that - based on share/functions/funced.fish.
-set -q TMPDIR
-or set -l TMPDIR /tmp
-set tmpdir (mktemp -d $TMPDIR/fish.XXXXXX)
-or exit 1
+set -g output_file (status dirname)/../messages.pot
+
+# Write header. This is required by msguniq.
+# Note that the first echo results in the file being overwritten.
+# This is desired behavior, to get rid of the results of prior invocations
+# of this script.
+echo 'msgid ""' > $output_file
```

Comment:
would be good to format the script fish with `fish_indent`

Reply:
If everyone agrees that we should use the formatter, I'd prefer that - it would be the progressive/modern approach.
But there are also good arguments against using automated code formatting (it strictly limits expressivity), and I don't really care